meta-lxatac-bsp/recipes-core/lxatac-net-switch-hacks/files/spi-irq-prio-44009000.service

```
@@ -0,0 +1,11 @@
+[Unit]
+Description=Increase kernel thread priority for the SPI bus to prevent timeouts
+Requires=sys-devices-platform-soc-5c007000.bus-44009000.spi.device
```

For devices I usually use `BindsTo`, but as this is a `Type=oneshot` service, this should not make a big difference.

```suggestion
BindsTo=sys-devices-platform-soc-5c007000.bus-44009000.spi.device
```

https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#BindsTo=

---

There is always something new to learn about systemd units, thanks!
I've updated the PR to use `BindsTo=`.